pkg/extensions/search/resolver.go

```diff
@@ -170,6 +171,7 @@ func (r *queryResolver) repoListWithNewestImage(ctx context.Context, store stora
 
 			imageSize := imageLayersSize + manifestSize + configSize
 
+			// Q why we don't handle this error?
 			imageConfigInfo, _ := olu.GetImageConfigInfo(repo, manifests[i].Digest)
```

Comment:
?

Reply:
I don't remember. We could add the error to the list the same as we do for line 193 for example. And break afterward.